lte/gateway/c/oai/lib/hashtable/obj_hashtable_uint64.c

```diff
@@ -994,6 +994,7 @@ hashtable_rc_t obj_hashtable_uint64_get_keys(
       }
     }
 
+    free(keysP);
```

Comment:
This function call is supposed to return the list of keys in a hashtable. If we free it here, then the function will always return an empty list. This memory should be freed by the caller.